speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racerunningmenus.cpp

```diff
@@ -41,6 +41,7 @@
 #include "legacymenu.h"
 #include "racescreens.h"
 
+#define FORCE_EXPERIMENT_SETUP
```

Comment:
You are now defining this in 2 places, meaning you need to remember to remove it in both. See if you can unify this.